packages/itwinui-css/src/information-panel/information-panel.scss

```diff
@@ -22,16 +22,16 @@
       opacity $iui-speed-fast ease;
   }
 
-  > .iui-resizer {
+  > .iui-table-resizer {
```

Comment:
What it has to do with table?

Reply:
Incidental issue when using find/replace. Fixed